src/instructlab/client.py

```diff
@@ -28,3 +32,31 @@ def list_models(
         return client.models.list()
     except OpenAIError as exc:
         raise ClientException(f"Connection Error {exc}") from exc
+
+
+def check_model(model, api_base: str, endpoint_url=None, http_client=None, ctx=None):
+    """Check if right model is selected or served."""
+    if endpoint_url:
+        api_base = endpoint_url
+    model_list = list_models(
+        api_base=api_base,
+        http_client=http_client,
+    )
+    if len(model_list.data) == 0:
+        raise ModelCheckException("No model is served")
+
+    model_ids = [m.id for m in model_list.data]
+    if model not in model_ids:
+        if endpoint_url:
+            # user not specify --model parameter
+            if model in (DEFAULT_MODEL, ctx.obj.config.chat.model):
+                msg = f"Model {model} is not served by the endpoint url {endpoint_url}. These are the served models: {model_ids}. Use '--model' to select right model to use."
```

Comment:
How about we just always use this message? It'll simplify the code and the extra instructions never hurt.